std_msgs/Header header
quad_ufabc/Vector3 attitude_error
quad_ufabc/Vector3 ang_vel_error